src/ApiPort.VisualStudio.Common/ViewModels/OptionsViewModel.cs

```diff
@@ -220,5 +262,33 @@ public bool HasError
 
             return targetInfos.Concat(uniqueFromTargetMap).ToList();
         }
+
+        private void Dispose(bool disposing)
```

Comment:
Autofac disposes of items that are registered to it when the container itself is disposed of.